Lib/fontTools/cffLib.py

```diff
@@ -414,6 +417,7 @@ def getItems(self, items, strings):
 
 
 class TopDictIndexCompiler(IndexCompiler):
+	maxOpStack = 193
```

Comment:
@miguelsousa where is this attribute used? why is it not not simply `maxOpStack = defaultMaxStack`?